zrml/swaps/src/weights.rs

```diff
@@ -94,11 +94,11 @@ impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
     // Storage: Tokens Accounts (r:46 w:22)
     // Storage: System Account (r:2 w:1)
     fn distribute_pool_share_rewards(a: u32, b: u32) -> Weight {
-        (70_035_000 as Weight)
-            // Standard Error: 283_000
-            .saturating_add((45_926_000 as Weight).saturating_mul(a as Weight))
-            // Standard Error: 283_000
-            .saturating_add((67_740_000 as Weight).saturating_mul(b as Weight))
+        (0 as Weight)
```

Comment:
This seems weird, this function wasn't modified; `a` is the number of accounts, so we don't have to worry about getting DDoS'ed, but this is curious.

Reply:
Yeah, nothing changed for `distribute_pool_share_rewards` since the last benchmark commit. The comparison to verify is [here](https://github.com/zeitgeistpm/zeitgeist/compare/ce520be36cc06d6687ca2f5c8499d2deb762d32d...e6ca87795f481728bd208487121a32f9ffb6c0d2?diff=split#diff-188fb32939fe9cc9416edd214b3ab316def04705fad5aa33b9130b8d49220244L938-L1061).

Reply:
This is out of my control, sometimes the benchmarks for the same benchmark logic start with a base weight of greater than zero and sometimes it's zero. I think as long as the weight is modified properly by variables this is acceptable - The weight calculated by the variables often overshadows the base weight by magnitudes anyways.
I would be altered though if the base weight is zero and isn't changed by anything else, like with `system.remark` in this PR.